workspaces/src/rpc/client.rs

```diff
@@ -333,15 +333,15 @@ impl Client {
 
 pub(crate) async fn access_key(
     client: &Client,
-    account_id: near_primitives::account::id::AccountId,
-    public_key: near_crypto::PublicKey,
+    account_id: &near_primitives::account::id::AccountId,
+    public_key: &near_crypto::PublicKey,
 ) -> Result<(AccessKeyView, CryptoHash)> {
     let query_resp = client
         .query(&methods::query::RpcQueryRequest {
             block_reference: Finality::None.into(),
             request: QueryRequest::ViewAccessKey {
-                account_id,
-                public_key,
+                account_id: account_id.clone(),
+                public_key: public_key.clone(),
```

Comment:
I don't see why this is changed. If this requires ownership, why not leave as an owned param and let the caller clone or not if needed?